website/addons/figshare/static/figshareFangornConfig.js

```diff
@@ -52,11 +62,12 @@ var _figshareItemButtons = {
             buttons.push(
                 m.component(Fangorn.Components.button, {
                     onclick: function(event) {
-                        Fangorn.ButtonEvents._gotoFileEvent.call(tb, item);
+                        window.open(item.data.extra.webView, '_self');
```

Comment:
Why not just make this an anchor tag rather than a button tag with `onclick`? That way, the user gets all the benefits of regular links (Cmd-click to open in new tab, right click to URL, show URL on hover, etc.)